typescript/sdk/src/deploy/verify/ContractVerifier.ts

```diff
@@ -92,6 +92,9 @@ export class ContractVerifier {
       params.set('apikey', apiKey);
     }
 
+    let timeout: number = 1000;
```

Comment:
this was intentional so adding Routescan or any others in the future is easy. preference thing i suppose

Reply:
this is a good thing for a comment because it is slightly unintuitive. a way to encode that optionality in code thought might be using a switch statement

Reply:
sure I'm keen. this will expand scope (if you look a few lines below, we have the same code smell), which i was trying to avoid by conforming to the function as it was.

regardless, will now go ahead and just update both branches